app/src/components/GrantCard.vue

```diff
@@ -0,0 +1,65 @@
+<!-- GrantCard -->
+
+<template>
+  <figure class="group">
+    <div class="relative cursor-pointer" @click="pushRoute({ name: 'dgrants-id', params: { id: id.toString() } })">
+      <img class="shadow-light" :src="imgurl" />
+      <div class="absolute bottom-0 right-0">
+        <!-- when item in cart, add class "in-cart" to <button> -->
+        <button
+          class="btn opacity-0 group-hover:opacity-100"
+          :class="{ 'in-cart': isInCart(id) }"
+          @click.stop="addToCart(id)"
+        >
+          <CartIcon />
+        </button>
+      </div>
+    </div>
+    <figcaption class="mt-4">
+      <div class="text-grey-500 font-medium truncate">{{ name }}</div>
+      <div class="flex justify-between">
+        <span class="text-grey-400"
+          >by
+          <a
+            class="text-grey-500 underline"
+            :href="`https://etherscan.io/address/${ownerAddress}`"
```

Comment:
This ok? Or should check the `network` we are in?

Reply:
Check out [getEtherscanUrl](https://github.com/dcgtc/dgrants/blob/126cd0ce9faf9d4e0a3c02d5083743166a9ac1b2/app/src/utils/utils.ts#L112) in utils - we could repurpose that fn to accept an additional prop for `tx`/`address`/`token` and that will handle the network part 🙌

Reply:
Check out getEtherscanUrl in utils - we could repurpose that fn to accept an additional arg for tx/address/token and that will handle the network part 🙌

Reply:
Oh nice @gdixon Something like this? https://github.com/dcgtc/dgrants/pull/184/commits/275c1c2c25332c7bc2b517ae62f179e6d73e0bdb

I used a third argument (`group`) with a default value to avoid breaking changes, but I could also update the calls to that function.